### DISCUSSION TITLE
Bot Not Responding

### DISCUSSION THREAD

**Discussion by Mehul343** [nextcord/nextcord#548] [Q&A]
My Discord bot stops responding after 1-2 hours. The bot is running and there are no errors in the console. Slash Commands seem to work but not the normal commands.

**Comment by TAG-Epic** [comment]
Can you send us a logfile? Info on how to obtain a log file can be found here https://docs.nextcord.dev/en/stable/logging.html

**Comment by ArjunSharda** [comment]
Do you use replit? Second, make sure you are using the correct prefix.

If you're using replit & uptimerobot, chances you're going to get ratelimited due to shared IP hosting. This is not on nextcord's end.

**Comment by ooliver1** [comment]
are prefix ('normal') commands working sometimes or just partially, what version and system info?

**Comment by acuimpact** [comment]


**Reply by ooliver1** [reply]
Please do not suggest using other libraries thanks

**Comment by ReaperZ0v** [comment]
when I am using a background task loop, I can't run a command while its running

how can i fix this?
